tests/src/test/java/alluxio/job/replicate/ReplicateIntegrationTest.java

```diff
@@ -76,4 +95,23 @@ public void replicateLastBlockFromUFS() throws Exception {
     Assert.assertEquals(TEST_BLOCK_SIZE, blockInfo1.getLength());
     Assert.assertEquals(1, blockInfo2.getLength());
   }
+
+  // Tests that we send one request and log one message when job master is at capacity
+  @Test
+  @LocalAlluxioClusterResource.Config(confParams = {PropertyKey.Name.JOB_MASTER_JOB_CAPACITY, "0",
```

Comment:
if 10 files need to be replicated, will the log message print 10 times?

Reply:
Can we test that it's no longer the case?

Reply:
My bad. I thought more files were created but it's two blocks..I adding more files for the test in next commit.